src/css/components/our-lessons.css

```diff
@@ -0,0 +1,174 @@
+:root {
+    font-family: "Manrope", sans-serif;
+}
+
+.section__title {
```

Comment:
Please take final version for "section-head" from Main branch.

Reply:
fixed